modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -16,6 +16,58 @@
 
 toc::[]
 
+== Infinite Scale 6.5.0 (Rolling)
+
+IMPORTANT: This is a Rolling Release. Please check the {release-types-url}[documentation] to see if this release type is right for your use case.
+
+Refer to the source and the full change log for a list of bug fixes and changes at {ocis-releases-url}/v6.5.0[GitHub, window=_blank].
+
+[discrete]
+=== Public and Private Links in the Sharing Panel
```

Comment:
You call them Public and Private in the heading, but the text uses Permanent and Public.

Edit the heading, or add some explanation that Private = Permanent